server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/dto/RegexMappingDTO.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.james.webadmin.dto;
```

Comment:
All our files are mandatory to start with the Apache V2 license. Can you add it?

Also a bit of IDE set up can be done to have it by default

Reply:
You resolved those but didn't solve them

Reply:
Sorry, I assumed that the license template was set up. Fixed.

Reply:
We need the license here anyway...